packages/fuel-gauge/src/doc-examples.test.ts

```diff
@@ -200,44 +200,6 @@ describe('Doc Examples', () => {
     // #endregion wallet-message-signing
   });
 
-  it('can create wallets', async () => {
```

Comment:
This test was used in the `test-wallets.md` documentation page and I removed it because I removed that page.